lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -209,28 +226,33 @@
     {{#modal.body}}
         {{project-selector
             user=user
-            newProject=newProject
-            file=selectedItems.firstObject
             projectSelectState=projectSelectState
             validationChanged=(action (mut isProjectSelectorValid))
-            projectSelected=(action (mut node))
+            projectSelected=(action projectSelected)
         }}
     {{/modal.body}}
     {{#modal.footer}}
-        {{#bs-button disabled=isMoving onClick=(action 'closeMoveToProjectModal') type="default"}}
+        {{#bs-button data-test-move-to-project-modal-close-button disabled=isMoving onClick=(action 'closeMoveToProjectModal') type="default"}}
             {{t 'general.cancel'}}
         {{/bs-button}}
         {{#if (not-eq projectSelectState 'main')}}
-            {{#bs-button disabled=isMoving onClick=(action (mut projectSelectState) 'main') type="default"}}
+            {{#bs-button data-test-move-to-project-modal-back-button disabled=isMoving onClick=(action (mut projectSelectState) 'main') type="default"}}
                 {{t 'general.back'}}
             {{/bs-button}}
-            {{#bs-button disabled=(or (not isProjectSelectorValid) isMoving) onClick=(action (perform moveToProject)) type="primary"}}
+            {{#bs-button data-test-move-to-project-modal-perform-button disabled=(or (not isProjectSelectorValid) isMoving) onClick=(action (perform moveToProject)) type="primary"}}
                 {{t 'file_browser.move_modal.move_button'}}
             {{/bs-button}}
         {{/if}}
     {{/modal.footer}}
 {{/bs-modal}}
-{{#bs-modal onHide=(action (mut currentModal) '') open=(eq currentModal 'successMove') class='successMoveQuickfileModal' as |modal|}}
+{{#if (eq currentModal 'successMove')}}
+    <NewProjectNavigationModal
```

Comment:
Ok, maybe `<NewProjectNavigationModal>` still needs to exist, but it could be used by `<NewProjectButton>`, hypothetically.

Reply:
Right, that's exactly the duplicate logic I'd want to get rid of. If it's implemented and well-tested one time in the component, we don't need to bother reimplementing it every time we add a "new project" button.